src/Model.elm

```diff
@@ -321,3 +231,72 @@ updateRecord formData record =
                     | title = Just formData.title
                     , description = Just formData.description
                 }
+
+
+
+-- Kinto client configuration
+
+
+client : Kinto.Client
+client =
+    Kinto.client
+        "https://kinto.dev.mozaws.net/v1/"
+        (Kinto.Basic "test" "test")
+
+
+recordResource : Kinto.Resource Record
+recordResource =
+    Kinto.recordResource "default" "test-items" decodeRecord
+
+
+decodeRecord : Decoder Record
+decodeRecord =
+    (map4 Record
+        (field "id" string)
+        (maybe (field "title" string))
+        (maybe (field "description" string))
+        (field "last_modified" int)
+    )
+
+
+
+-- Kinto API calls
+
+
+fetchRecord : RecordId -> Cmd Msg
+fetchRecord recordId =
+    client
+        |> Kinto.get recordResource recordId
+        |> Kinto.send FetchRecordResponse
+
+
+fetchRecords : Cmd Msg
```

Comment:
Nit: `fetchRecordList` maybe, as `fetchRecords` is visually harder to distinguish from `fetchRecord` at a quick glance